nano/node/active_transactions.cpp

```diff
@@ -591,6 +674,24 @@ size_t nano::active_transactions::size ()
 	return roots.size ();
 }
 
+size_t nano::active_transactions::long_unconfirmed_size ()
```

Comment:
Have to iterate over all roots each time. Probably variable can stored from nano::active_transactions::request_confirm (), unconfirmed_count? And then updated in nano::election::confirm_once () if announcements > announcement_long